app/src/main/java/io/multy/ui/fragments/seed/SeedValidationFragment.java

```diff
@@ -116,9 +119,40 @@ public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
             public void afterTextChanged(Editable editable) {
                 if (editable.toString().length() == 0) {
                     inputWord.setGravity(Gravity.LEFT);
+                    buttonNext.setText(R.string.next_word);
+                    return;
                 } else if (editable.toString().length() == 1) {
                     inputWord.setGravity(Gravity.CENTER_HORIZONTAL);
                 }
+                if (editable.length() > 0) {
```

Comment:
maybe we need to start making possible solution only after 1 or 2 symbol?

Reply:
Nope. That's fine!